tests/switchmap_/server/db/ingest/test_ingest.py

```
@@ -171,47 +293,294 @@ def tearDown(cls):
+            if os.path.exists(skip_path):
+                os.remove(skip_path)
+
+    def test_setup_function_returns_event_objects(self):
+        """Test that setup returns a valid EventObjects instance with zones."""
+        src = "tests/testdata_"  # Folder containing device YAML files
+
+        result = ingest.setup(src, self.config)
+
+        self.assertIsInstance(result, EventObjects)
+        self.assertTrue(len(result.zones) > 0)
+        for zone_item in result.zones:
```

Ruff (0.8.2)

Loop control variable `zone_item` not used within loop body

Rename unused `zone_item` to `_zone_item`

(B007)